l10n_ch_account_statement_base_import/data/UBS_export.csv

```diff
@@ -7,4 +7,5 @@ Date d'
 31.08.2014;0298 00101368;;0298 00101368.01Q;CH08 0029 8298 9999 9901 Q;CHF;01.08.2014;31.08.2014;UBS Current Account for Private Clients;10.08.2014;31.08.2014;10.08.2014;Dividend;VN   472672     101368S3;NOKIA;BC80538KE0773604;;;;76.00;20'365.60
 
 Solde final;Solde initial
-20827.000000000;20289.600000000
\ No newline at end of file
+20827.000000000;20289.600000000

```

Comment:
Here I definitely think the error is not this file but the lint tool. I wonder if this caused the travis error:

2016-07-29 07:18:29,182 24073 ERROR openerp_test openerp.addons.l10n_ch_account_statement_base_import.tests.test_ubs: ` openerp.exceptions.Warning: The following problem occurred during ubs_csvparser import. The file might not be valid.


list index out of range might be due to the new line as parser tries to read an additional line?